src/EnergyPlus/ZoneDehumidifier.hh

```diff
@@ -63,142 +63,108 @@ struct EnergyPlusData;
-        int HighPLFErrorCount;             // Count number of times PLF > 1.0
-        int HighPLFErrorIndex;             // Index for PLF > 1.0 recurring warning messages
-        int HighRTFErrorCount;             // Count number of times RTF > 1.0
-        int HighRTFErrorIndex;             // Index for RTF > 1.0 recurring warning messages
-        int PLFPLRErrorCount;              // Count number of times PLF < PLR
-        int PLFPLRErrorIndex;              // Index for PLF < PLR recurring warning messages
-        int CondensateCollectMode;         // Where does water come from
-        std::string CondensateCollectName; // Name of water storage (collection) tank
-        int CondensateTankID;              // Condensate collection tank ID number
-        int CondensateTankSupplyARRID;     // Condensate collection tank supply ID number
+        std::string Name;                                                     // Name of unit
+        std::string UnitType;                                                 // Type of unit
+        int SchedPtr = 0;                                                     // Index number to availability schedule
+        Real64 RatedWaterRemoval = 0.0;                                       // Rated water removal [liters/day]
+        Real64 RatedEnergyFactor = 0.0;                                       // Rated energy factor [liters/kWh]
+        Real64 RatedAirVolFlow = 0.0;                                         // Rated air flow rate through the dehumidifier [m3/s]
+        Real64 RatedAirMassFlow = 0.0;                                        // Rated air mass flow rate through the dehumidifier [kg/s]
+        Real64 MinInletAirTemp = 0.0;                                         // Minimum dry-bulb temperature for dehumidifier operation [C]
+        Real64 MaxInletAirTemp = 0.0;                                         // Maximum dry-bulb temperature for dehumidifier operation [C]
+        Real64 InletAirMassFlow = 0.0;                                        // Inlet air mass flow rate for the time step being simulated [kg/s]
+        Real64 OutletAirEnthalpy = 0.0;                                       // Dehumidifier outlet air enthalpy [J/kg]
+        Real64 OutletAirHumRat = 0.0;                                         // Dehumidifier outlet air humidity ratio [kg/kg]
+        Real64 OffCycleParasiticLoad = 0.0;                                   // Off Cycle Parasitic Load, user input [W]
+        int AirInletNodeNum = 0;                                              // Inlet air node number
+        int AirOutletNodeNum = 0;                                             // Outlet air node number
+        int WaterRemovalCurveIndex = 0;                                       // Index for water removal curve
+        int WaterRemovalCurveErrorCount = 0;                                  // Count number of times water removal curve returns a negative value
```

Comment:
This repeating `ErrorCount`, `ErrorIndex` pair pattern should be condensed to a struct at some point.